src/rejection_intake.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+if (!defined('not_direct_access') && !empty($_POST['pre']) && !empty($_POST['post']) && !empty($_REQUEST['hunter_id_hash'])) {
+    define('not_direct_access', TRUE);
+    require_once "check-cors.php";
+    require_once "uuid.php";
```

Comment:
you mentioned to require_once check-version.php in discord. just making this not not to forget.